### PR TITLE
Rename pulp services

### PR DESCRIPTION
**Pull request by lubosmj**
closes #4554
https://pulp.plan.io/issues/4554

### CODE REVIEW

**Comment by gmbnomis** [comment]
@lubosmj Thank you for providing the PR. This depends on https://github.com/pulp/ansible-pulp/pull/161, right? (or are there other dependencies as well?)

**Comment by lubosmj** [comment]
Yes, it could depend on https://github.com/pulp/ansible-pulp/pull/161 and on https://github.com/pulp/pulpcore/pull/307 as well. Also, I have sent an e-mail to pulp-dev mailing list which highlights the changes. You might find it helpful.

File: CHANGES/4554.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Change the prefix of Pulp services from pulp-* to pulpcore-*
```
Comment (gmbnomis) [inline CHANGES/4554.misc]: This plugin tracks its issues on GitHub and not on pulp.plan.io, I have created an issue (https://github.com/gmbnomis/pulp_cookbook/issues/49) for this change. Could you please rename the file to `49.misc`.

File: .travis/script.sh [hunk 2]
```diff
@@ -112,7 +112,7 @@ show_logs_and_return_non_zero() {
 }
 
 # Stop services started by ansible roles
-sudo systemctl stop pulp-worker* pulp-resource-manager pulp-content-app pulp-api
```
Comment (gmbnomis) [inline .travis/script.sh]: In the meantime, I have updated the Travis config to use the new (Docker-based) CI runs. I think you can simply drop the change to this file.

**Pull request merged**